### PR TITLE
many: replace use of "sanity" with more inclusive naming (part 2)

### PR DESCRIPTION
**Pull request by mvo5**
This commit replaces the use of "sanity" with more inclusive
naming. When "sanity" is used in a more general sense either
"validity" or "quick" is used.

### CODE REVIEW

**Review by MiguelPires — APPROVED** [review]
LGTM

File: cmd/autogen.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ if [ ! -e VERSION ]; then
 	( cd .. && ./mkversion.sh )
 fi
 
-# Sanity check, are we in the right directory?
+# Validity check, are we in the right directory?
```
Comment (pedronis) [inline cmd/autogen.sh:17]: I think in this kind of case and similar,  precodintion check would be a clearer/better term

File: interfaces/apparmor/backend_test.go [hunk 2]
```diff
@@ -1422,7 +1422,7 @@ func (s *backendSuite) TestSnapConfineProfileDiscardedLateSnapd(c *C) {
 	s.writeVanillaSnapConfineProfile(c, snapdInfo)
 	err := s.Backend.Setup(snapdInfo, interfaces.ConfinementOptions{}, s.Repo, s.perf)
 	c.Assert(err, IsNil)
-	// sanity
+	// validity
```
Comment (stolowski) [inline interfaces/apparmor/backend_test.go:1425]: Precondition?

File: image/image_test.go [hunk 3]
```diff
@@ -678,7 +678,7 @@ func (s *imageSuite) TestSetupSeed(c *C) {
 
 			Channel: stableChannel,
 		})
-		// sanity
+		// validity
```
Comment (stolowski) [inline image/image_test.go:681]: Precondition check?

File: interfaces/repo.go [hunk 4]
```diff
@@ -645,7 +645,7 @@ func (r *Repository) Disconnect(plugSnapName, plugName, slotSnapName, slotName s
 	r.m.Lock()
 	defer r.m.Unlock()
 
-	// Sanity check
+	// Validity check
```
Comment (stolowski) [inline interfaces/repo.go:648]: Same.

File: interfaces/system_key.go [hunk 5]
```diff
@@ -299,7 +299,7 @@ func CurrentSystemKey() (interface{}, error) {
 
 // SystemKeysMatch returns whether the given system keys match.
 func SystemKeysMatch(systemKey1, systemKey2 interface{}) (bool, error) {
-	// sanity check
+	// validity check
```
Comment (stolowski) [inline interfaces/system_key.go:302]: Same.

**Review by stolowski — COMMENTED** [review]
Thanks for doing this! I agree with Samuele, in a few cases "precondition" would suit better I think.

**Comment by mvo5** [comment]
Thanks for the suggestions! Updated this now

**Review by mardy — APPROVED** [review]
LGTM!

**Pull request merged**